mos.yml

```diff
@@ -29,6 +29,8 @@ config_schema:
   - ["sw.in_mode", "i", 1, {"0 - Momentary, 1 - Toggle, 2 - Edge, 3 - Detached"}]
   - ["sw.state", "b", false, {"State of the switch"}]
   - ["sw.persist_state", "b", false, {"Whether state of the switch should be persisted across reboots."}]
+  - ["sw.auto_off", "b", false, {"Whether the switch should automatically turn OFF after turning ON"}]
+  - ["sw.auto_off_delay", "s", "", {"Delay for automatically turning OFF, in the 24 hour format HH:mm:ss."}]
```

Comment:
i'd rather we keep string conversion operations in JS and make this an int variable